closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -3496,7 +3508,7 @@ setECPublicKey(EC_KEY *key, BIGNUM *x, BIGNUM *y, int field)
  */
 JNIEXPORT jint JNICALL
 Java_jdk_crypto_jniprovider_NativeCrypto_PBEDerive
```

Comment:
Please correct the signature in the comment on line 3507 (there a no `long` parameters):
```c
 * Signature: ([BI[BI[BIIII)I
```